src/apps/api/views/competitions.py

```diff
@@ -96,6 +102,41 @@ def get_files(self, request, pk):
         return Response(files)
 
 
+@api_view(['GET'])
+@permission_classes((AllowAny,))
+def front_page_competitions(request):
+    popular_comps = get_popular_competitions()
+    featured_comps = get_featured_competitions(excluded_competitions=popular_comps)
+    popular_comps_serializer = CompetitionSerializerSimple(popular_comps, many=True)
+    featured_comps_serializer = CompetitionSerializerSimple(featured_comps, many=True)
+    return Response(data={
+        "popular_comps": popular_comps_serializer.data,
+        "featured_comps": featured_comps_serializer.data
+    }, status=status.HTTP_200_OK)
+
+
+@api_view(['GET'])
+@permission_classes((AllowAny,))
+def by_the_numbers(request):
+    total_competitions = Competition.objects.all().count()
```

Comment:
Can save some queries by doing:
```python
data = Competition.objects.aggregate(
    count=Count('*'), 
    published_comps=Count('pk', filter=Q(published=True)),
    unpublished_comps=Count('pk', filter=Q(published=False)), 
    organizers=Count('created_by', distinct=True)
)
# returns a dict that looks like: {'count': 31, 'published_comps': 20, 'unpublished_comps': 11, 'organizers': 12}
total_competitions = data['count']
public_competitions = data['published_comps']
private_competitions = data['unpublished_comps']
organizers = data['organizers']
```
Turns 4 queries into 1

Reply:
Fixed in latest push, removed organizers for now because seven queries looked unbalanced on the front page. We can pull a different category later if necessary for Organizers. If we pull private competitions but leave Public and Total, we have an implicit count of Private Competitions so maybe a good choice.